service/templates/distros.html

```diff
@@ -424,11 +424,25 @@ <h2 style="display:inline-block; padding-right:15px">Configure
                 <div class="icon fa fa-warning distro-error" ng-show="form.keyName.$dirty && form.keyName.$error.required || form.keyName.$invalid">EC2
                   key name is required</div>
               </div>
-              <div class="distro-label">
-                <div ng-show="activeDistro.provider == 'ec2-spot' || activeDistro.provider == 'ec2-fleet'">
-                  <label><input style="margin-right:10px;" ng-disabled="readOnly" type="checkbox"
-                    name="fallback" ng-model="activeDistro.settings.fallback">Fallback to EC2 On-Demand in the case of insufficient capacity </label> <br>
+              <div ng-show="activeDistro.provider === 'ec2-spot'">
```

Comment:
can you please include a screenshot of the visible changes?

Reply:
done